kolibri/plugins/facility/assets/src/views/UserPage/index.vue

```diff
@@ -72,55 +131,82 @@
           @changeSort="changeSortHandler"
         >
           <template #header="{ header, colIndex }">
-            <span :class="{ visuallyhidden: colIndex === 5 }">{{ header.label }}</span>
-            <span v-if="colIndex === 2">
-              <CoreInfoIcon
-                class="tooltip"
-                :iconAriaLabel="coreString('identifierAriaLabel')"
-                :tooltipText="coreString('identifierTooltip')"
-              />
-            </span>
+            <template v-if="colIndex === 0">
+              <div :aria-label="selectLabel$()">
+                <KCheckbox
+                  :label="selectAllLabel$()"
+                  :checked="selectAllState.checked"
+                  :indeterminate="selectAllState.indeterminate"
+                  :showLabel="false"
+                  :ariaLabel="selectAllLabel$()"
+                  @change="handleSelectAllToggle"
+                />
+              </div>
+            </template>
+            <template v-else>
+              <span :class="{ visuallyhidden: colIndex === 7 }">{{ header.label }}</span>
+              <span v-if="colIndex === 3">
+                <CoreInfoIcon
+                  class="tooltip"
+                  :iconAriaLabel="coreString('identifierAriaLabel')"
+                  :tooltipText="coreString('identifierTooltip')"
+                />
+              </span>
+            </template>
           </template>
+
           <template #cell="{ content, colIndex, row }">
-            <span v-if="colIndex === 0">
+            <div
+              v-if="colIndex === 0"
+              :aria-label="selectLabel$()"
+            >
+              <KCheckbox
+                :checked="isUserSelected(row[6])"
+                :showLabel="false"
+                :aria-label="row[1] + ',' + row[6].kind"
```

Comment:
Right now we're passing the `.kind` string directly to the aria label. But, we need to make sure that we are passing a translated string. It should match the label for the name in all cases (i.e. follow these conventions for admin, super admin, coach types, etc.) and then add in "learner" 

<img width="189" alt="Screenshot 2025-06-16 at 6 47 07 PM" src="https://github.com/user-attachments/assets/36e9b05a-028f-454a-bbbd-f9e210aa3f30" />